x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_configuration.tsx

```diff
@@ -141,7 +141,7 @@ export const ConnectorConfiguration: React.FC = () => {
                           values={{
                             link: (
                               <EuiLink
-                                href="https://github.com/elastic/connectors-ruby/tree/main/config"
+                                href="https://github.com/elastic/connectors-python/tree/blob/config.yml"
```

Comment:
This 404ed for me, but https://github.com/elastic/connectors-python/blob/main/config.yml worked

Reply:
```suggestion
                                href="https://github.com/elastic/connectors-python/blob/main/config.yml "
```